libs/designsystem/src/lib/components/form-field/form-field.component.ts

```diff
@@ -1,45 +1,67 @@
 import {
+  AfterContentChecked,
+  AfterContentInit,
   ChangeDetectionStrategy,
   Component,
-  Input,
   ContentChild,
-  AfterContentChecked,
-  OnDestroy,
   ElementRef,
-  OnInit,
   HostListener,
+  Input,
+  OnDestroy,
+  OnInit,
+  Renderer2,
 } from '@angular/core';
 
 import { PlatformService } from '../../helpers/platform.service';
 import { WindowRef } from '../../types/window-ref';
+import { RadioGroupComponent } from '../radio';
 
 import { InputCounterComponent } from './input-counter/input-counter.component';
+import { InputComponent } from './input/input.component';
+import { TextareaComponent } from './textarea/textarea.component';
+
+// Counter for generating unique element ids
+let uniqueId = 0;
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'kirby-form-field',
   styleUrls: ['./form-field.component.scss'],
   templateUrl: './form-field.component.html',
 })
-export class FormFieldComponent implements AfterContentChecked, OnInit, OnDestroy {
+export class FormFieldComponent
+  implements AfterContentChecked, AfterContentInit, OnInit, OnDestroy {
   private isRegistered = false;
   private element: HTMLElement;
   private inputElement: HTMLInputElement | HTMLTextAreaElement;
   private isTouch: boolean;
+  _labelId = `kirby-form-field-label-${uniqueId++}`;
 
   @Input() label: string;
   @Input() message: string;
 
   @ContentChild(InputCounterComponent, { static: false }) counter: InputCounterComponent;
+  @ContentChild(RadioGroupComponent) private radioGroupComponent: RadioGroupComponent;
+  @ContentChild(RadioGroupComponent, { read: ElementRef }) private radioGroupElement: ElementRef<
+    HTMLElement
+  >;
+
+  @ContentChild(InputComponent, { read: ElementRef }) input: ElementRef<HTMLInputElement>;
+  @ContentChild(TextareaComponent, { read: ElementRef }) textarea: ElementRef<HTMLTextAreaElement>;
 
   constructor(
     elementRef: ElementRef<HTMLElement>,
     private platform: PlatformService,
+    private renderer: Renderer2,
     private window: WindowRef
   ) {
     this.element = elementRef.nativeElement;
   }
 
+  get _wrapContentInLabel(): boolean {
```

Comment:
Why the _ notation? Usually something we did on the old days of JS to indicate a private property, but now we have the type system for that?

Reply:
The _ properties seem scattered all over this PR

Reply:
@jb-roa We use this notation to express the intent that this method is "internal", i.e. shouldn't be used although it's part of the public API. It's the only way we can do this, as the template needs public access to the method.